index.markdown

```diff
@@ -135,10 +136,13 @@ start date, and if they're using Cultural Vistas as a visa sponsor. It's useful
 for the visa. If your company isn't using Cultural Vistas, good luck, and be
 ready to provide a lot of school-related paperwork such as proof of enrolment.
 
-CECA and Cultural Vistas will guide you through the process.
+CECA and Cultural Vistas will guide you through the process. If you're an international
+student (i.e your passport isn't Canadian), this process will involve making a trip
```

Comment:
s/If you're an international student (i.e your passport isn't Canadian),/If you are not a Canadian citizen/